action.yml

```diff
@@ -56,3 +56,11 @@ inputs:
       Also when generating a new PAT, select the least scopes necessary.
       [Learn more about creating and using encrypted secrets](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-and-using-encrypted-secrets)
     default: ${{ github.token }}
+
+outputs:
+  ssh-command:
+    description: 'The SSH command to connect to the tmate session (only set when detached mode is enabled)'
+  ssh-address:
+    description: 'The raw SSH address without the "ssh" prefix (only set when detached mode is enabled)'
+  web-url:
+    description: 'The web URL to connect to the tmate session (only set when detached mode is enabled and web URL is available)'
```

Comment:
Please also add this to [`detached/action.yml`](https://github.com/mxschmitt/action-tmate/blob/HEAD/detached/action.yml).